recipe/meta.yaml

```diff
@@ -13,10 +13,9 @@ source:
   url: https://github.com/quarto-dev/quarto-cli/releases/download/v{{ version }}/quarto-{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-  - 0001-look-for-tinytex-in-conda-bin-paths-when-CONDA_PREFI.patch
-  - 0001-win-conda-path-overrides.patch
-  - 0001-respect-QUARTO_PANDOC-in-update-pandoc.patch
```

Comment:
The `update_pandoc` component of Quarto seems to have been updated so that instead of running pandoc to generate template files, it's now extracting them from source.

https://github.com/quarto-dev/quarto-cli/compare/v1.2...v1.3.340#diff-133d2fed884afdcf09a58b7a5a8edb21a04360dae8ad0a73166c61a506a5ba89R184-R186